docs/developer.md

```diff
@@ -36,6 +36,18 @@ For an in-depth look at MDC-Web's architecture, check out [architecture.md](./ar
 
 Check out our `webpack.config.js` for more details!
 
+### Deployment
+
+Demo pages of Material Components for Web are deployed in Google Cloud. Request an access to project id: material-components-web to get deployment permissions to app engine.
```

Comment:
I'd probably take out the part regarding requesting access. It might be more appropriate within `CONTRIBUTING.md` under the Releasing section. Furthermore, we'll want to put instructions within the CONTRIBUTING docs on what `gcloud` is and how to install it.